src/loggerMiddleware.d.ts

```diff
@@ -0,0 +1,10 @@
+import Rodux from "@rbxts/rodux";

```

Comment:
A package importing itself by its org name does not make sense. You can take a look at how https://github.com/roblox-ts/rbx-rodux/blob/master/src/Store.d.ts does it.